src/components/Rocket.jsx

```diff
@@ -4,50 +4,58 @@ import { useSelector, useDispatch } from 'react-redux';
 import { reserveRocket, cancelRocket } from '../redux/rocketsReducer';
 import './css/Rocket.css';
 
-Rocket.propTypes = {
-    rocket: PropTypes.shape({
-        id: PropTypes.number.isRequired,
-        rocket_name: PropTypes.string.isRequired,
-        description: PropTypes.string.isRequired,
-        flickr_images: PropTypes.arrayOf(PropTypes.string).isRequired,
-    }).isRequired,
-};
-
 function Rocket({ rocket }) {
   const rockets = useSelector((state) => state.rockets.rockets);
-  const { id, rocket_name, description, flickr_images,  } = rocket;
-  const [image, setImage] = useState(flickr_images[0]);
-    const dispatch = useDispatch();
+  const {
+    // eslint-disable-next-line camelcase
+    id, rocket_name, description, flickr_images,
+  } = rocket;
+  // eslint-disable-next-line camelcase
```

Comment:
Something else to note, instead of having many such comments on multiple lines, I think it would be better if you disabled this linter error for the entire file.